tests/integration/test_enrollment/data/wazuh_enrollment_tests.yaml

```diff
@@ -0,0 +1,273 @@
+-
+  name: "Default Name"
+  description: "No agent name in auto enrollment configuration"
+  pre_existent_keys:
+    - ""
+  message:
+    expected: "OSSEC A:'{host_name}'\n"
+    response: "OSSEC K:'001 {host_name} any TopSecret'\n"
+-
+  name: "Valid agent address"
+  description: "Check a valid agent_address configuration"
+  configuration:
+    agent_address: "192.168.0.10"
+  pre_existent_keys:
+    - ""
+  message:
+    expected: "OSSEC A:'{host_name}' IP:'192.168.0.10'\n"
+    response: "OSSEC K:'001 {host_name} 192.168.0.10 TopSecret'\n"
+-
+  name: "Valid agent address CIDR"
+  description: "Check a valid agent_address configuration CIDR"
+  configuration:
+    agent_address: "192.168.0.10/24"
+  pre_existent_keys:
+    - ""
+  message:
+    expected: "OSSEC A:'{host_name}' IP:'192.168.0.10/24'\n"
+    response: "OSSEC K:'001 {host_name} 192.168.0.10/24 TopSecret'\n"
+-
+  name: "Invalid Agent address CIDR"
+  description: "Check an invalid CIDR agent_address in auto enrollment configuration"
+  configuration:
+    agent_address: "192.168.0.10/100"
+  expected_error: "ERROR: (4104): Invalid hostname: '192.168.0.10/100'."
+  #Known Issue: Invalid CIDR is not checked. Skiped
+  skips:
+    - agent-auth
+    - wazuh-agentd
```

Comment:
If there is a known issue, we can't `skip` or `xfail` the tests. Also, the issue should be open and indicated.

Reply:
Fixed

Reply:
The test isn´t skiped any more. It's now expecting the error.